tests/runtests.bats

```diff
@@ -223,3 +223,64 @@ cleanup() {
 
   cleanup
 }
+
+@test "bring up and save a snapshot" {
+  export VAGRANT_CWD=tests/snapshot
```

Comment:
I think maybe skip this test as you end up testing the same thing in the following one. Unless there was something specific for the snapshot saving that needed to be tested here or due to performance when testing locally that it was useful to test save by itself.

Reply:
Actually, there is no hidden purpose here. I was trying to test that those three operations were working properly. But I can remove it, since the full snapshot test I will do most of the operations here.